src/linker/Linker/PInvokeInfo.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Runtime.Serialization;
+
+namespace Mono.Linker
+{
+	[DataContract]
+	public class PInvokeInfo
+	{
+		[DataMember (Name = "entryPoint")]
+		internal string EntryPoint { get; set; }
+		
+		[DataMember (Name = "moduleName")]
+		internal string ModuleName { get; set; }
+	}
```

Comment:
We also need to know:

* The assembly the symbol came from. This is used to select whether we ask the native linker to reference the symbol or not.
* The full name of the C# P/Invoke. This is used when showing an error message if the native linker can't find a particular symbol.